functions.php

```diff
@@ -37,3 +37,9 @@ function( $template ) {
 
 require_once __DIR__ . '/classes/class-p4-loader.php';
 P4_Loader::get_instance();
+
+// Needed to allow the editor rule to change the author of a post in the document sidebar. The users data for that
+// control is fetched using the REST API, where WordPress by default doesn't perform a permissions check, however
+// the Wordfence plugin adds this check in `\wordfence::jsonAPIAuthorFilter`.
+$roles = wp_roles();
```

Comment:
we may add this piece of code at [here](https://github.com/greenpeace/planet4-master-theme/blob/develop/classes/class-p4-activator.php#L32) in function

Reply:
Doesn't that only get run when the theme is switched? see https://github.com/greenpeace/planet4-master-theme/blob/develop/classes/class-p4-activator.php#L25

Reply:
Yes it execute on theme switch as well as once on every deploy (in post deploy execution script) [https://github.com/greenpeace/planet4-base-fork/blob/develop/tasks/post-deploy/01-run-p4-activator.sh#L8]

More details about the post deploy P4 activator class is in [PLANET-4050](https://jira.greenpeace.org/browse/PLANET-4050)